packages/shared/__tests__/describeComponentFrame-test.js

```diff
@@ -11,13 +11,19 @@
 
 let React;
 let ReactDOM;
+const ReactFeatureFlags = require('shared/ReactFeatureFlags');
 
 describe('Component stack trace displaying', () => {
   beforeEach(() => {
     React = require('react');
     ReactDOM = require('react-dom');
   });
 
+  if (ReactFeatureFlags.enableComponentStackLocations) {
```

Comment:
This doesn't look necessary (at least not yet?)

Reply:
I think I'll just remove this test later. It's not very useful for this model. This test fails with the stack on, partly because the warning gets deduped because it's always the same component and line being called.